index.js

```diff
@@ -209,9 +222,26 @@
           }
           return isPropertyDefined('name', config) ? (config.name + '.' + commandName) : commandName;
         };
-
+        
+        var routes = readFromRoute ? $injector.get('$route').routes : { };
```

Comment:
This is assumptive that ng-route is actually defined and will throw a null pointer exception if not. It would be better (as in more helpful to a user) to check that $route exists and has a 'routes' property. If either of those conditions don't exist then output a $log.warn message.

```javascript
var routes = {};
if (readFromRoute) {
  var $route = $injector.get('$route');
  if (!$route) {
    $log.warn('$route service is not available. Make sure you have included ng-route in your application dependencies.');
  } else {
    routes = $route.routes;
  }
}
```